.nvmrc

```diff
@@ -0,0 +1 @@
+20.13.1
```

Comment:
do we want local locked?

Reply:
Yes, it matches what we're doing elsewhere.